test/unit/Util/Autoload/ClassLoaderTest.php

```diff
@@ -72,19 +75,18 @@ public function testAddClassThrowsExceptionWhenClassAlreadyRegisteredInAutoload(
 
         $this->expectException(ClassAlreadyRegistered::class);
         $loader->addClass($reflection);
-
-        spl_autoload_unregister($loader);
```

Comment:
Probably to be kept in the same process?

Reply:
It's not called because of thrown exception.